_posts/2016-09-22-professional-development-at-artsy-engineering.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "Professional Development at Artsy Engineering"
+date: 2016-09-22 16:37
+comments: true
+author: alan
+categories: [engineering, professional, development, organization, education]
+---
+
+In considering an offer to join us at Artsy, one of our newest incoming engineers asked me a great question: _How does the tech team do professional development?_
+
+As I thought about it, I began to realize that the answer is “a lot”! Most of our efforts evolved organically. Someone had an idea, and people rallied around it. I thought it would be useful to share, in case others are inspired by what's caught on here. Here are some of the things we do.
+
+<!-- more -->
+
+# Practices
+
+Our engineering team used to be organized along _practice_ lines: web, mobile native, and platform. Following our recent [product reorganization][1], we’re primarily organized in product teams, which are more deeply integrated and aligned with our business units. But practices are here to stay. They serve as horizontal channels for developers working in similar technologies to collaborate and share best practices. They each have a lively Slack channel and do standups once or twice a week.
+
+# Product teams
+
+Within product teams, engineers take on work across the stack. Most engineers have a comfort zone, and it's largely up to individuals to decide when and how to branch out. This gives people prerogative in shaping their expertise in accordance with their goals as a developer.
```

Comment:
From an outsider's perspective, this might sound like a completely hands-off approach when, in reality, product team leads are responsible for providing guidance and for nudging individual contributors to grow. Could use some clarification here.